test/E2E/TestFunctionApp/profile.ps1

```diff
@@ -0,0 +1,22 @@
+# Azure Functions profile.ps1
```

Comment:
Is this auto-generated? Unless you really need it, I suggest you exclude it from the PR